jerry-core/ecma/operations/ecma-regexp-object.c

```diff
@@ -1148,7 +1148,7 @@ re_match_regexp (re_matcher_ctx_t *re_ctx_p, /**< RegExp matcher context */
       }
       default:
       {
-        JERRY_TRACE_MSG ("UNKNOWN opcode (%d)!\n", (uint32_t) op);
+        JERRY_TRACE_MSG ("UNKNOWN opcode (%" PRIu8 ")!\n", op);
```

Comment:
`op` is of type `re_opcode_t`, which is an enum, thus its size is implementation specific. Therefore `op` should be cast to `uint8_t` here to be portable.